website/static/files/build_orca_inventory_refined.sql

```diff
@@ -123,6 +126,8 @@ BEGIN
       IS 'Cumulus S3 bucket where the file is thought to be stored.';
     COMMENT ON COLUMN files.key_path
       IS 'Full AWS key path including file name of the file (does not include bucket name) where the file resides in ORCA.';
+    COMMENT ON COLUMN files.multipart_chunksize_mb
```

Comment:
This comment I believe is in the wrong place maybe or an oversight? The column `multipart_chunksize_mb` does not and should not exist in this table.

Reply:
got confused with `multipart_chunksize_mb` being added to granules table in the schema for some reason.